### ISSUE TITLE
Lessons are not working

### ISSUE THREAD

**Issue by kanevk** [scdekov/memword#9]
I'm trying to do a lesson, but I cannot proceed with the next word.
**Clicking of neither of the buttons moves me forwards**

<img width="1440" alt="screen shot 2019-01-17 at 10 51 38 pm" src="https://user-images.githubusercontent.com/8972876/51348116-89319000-1aaa-11e9-8701-819b7fe5f457.png">

**Comment by scdekov** [comment]
Good catch! I messed this up when implementing the exams. It was a simple typo in a variable name, so I fixed it and deployed.
